indico/modules/events/abstracts/forms.py

```diff
@@ -390,14 +390,25 @@ class CreateEmailTemplateForm(EditEmailTemplateRuleForm):
                      "You can customize it later."))
 
 
-class AbstractForm(IndicoForm):
+class AbstractAttachmentMixin(object):
+    attachments = FileField(_('Attachments'), param_name='attachments', multiple_files=True)
```

Comment:
Now the one used for abstract submission is not lightweight anymore. Is that intentional? If not maybe it's better to just keep the fields separate instead of using a mixin.